demo/src/App.tsx

```diff
@@ -34,44 +35,45 @@ export default function App() {
 
     return (
         <>
```

Comment:
You can add an outer `Container` here to ensure horizontal centering. You can also add some padding here (like what I think you had before), but don't pass in css styles in the `sx` prop, instead use Material UI's system props (https://mui.com/system/properties/).

Reply:
I believe that when we wrap the component in `<main className="container">`, the styling adds horizontal centering (see screenshot below). Please let us know if the using the pre-existing styling does not to guarantee horizontal centering.

<details>
<summary>Screenshot</summary>

![image](https://github.com/user-attachments/assets/fff06f63-0a74-4b42-8373-aba2643df551)

</details>

Reply:
okay great, no need to add the `Container` element then 👍